### ISSUE TITLE
Bot with proxies not working

### ISSUE THREAD

**Issue by Yarumina** [HerrEurobeat/steam-comment-service-bot#147]
Hey Mate :)

i try to use the bot with proxies but it says the same thing all the time and is standing still. See the pic
![asdasd](https://user-images.githubusercontent.com/78469667/209453775-c6e65b45-dc3b-467e-b38f-be9931f44fa2.PNG)

So is it because the proxies are not working or why? also how is the best method to use proxies

**Comment by Yarumina** [comment]
Is there any chance  that you explain how to set up proxies right with the bot?

**Comment by Yarumina** [comment]
So i got it work kinda but the bots are keep disconnecting because of No Connection as a error

**Comment by HerrEurobeat** [comment]
Hey!  
The issue you've encountered above might happen when the bot couldn't connect to Steam with the proxy. The login itself doesn't use proxies (yet), that's why the login itself did work however.  
I've tried to explain everything there is to know about setting up proxies [here in the wiki](https://github.com/HerrEurobeat/steam-comment-service-bot/wiki/Adding-proxies) - did you read it? If there is something missing or unclear let me know :D  
  
A regularly recurring `NoConnection` error might be caused by a proxy with an unstable internet connection; I don't know if there is much you can do about it except for trying a different one.

**Comment by Yarumina** [comment]
I tried to do it like in the wiki, but it never worked. But when i just add the proxies everything is working but the errors are coming. What proxy provider do you use?

**Comment by HerrEurobeat** [comment]
I use webshare.io

**Comment by Yarumina** [comment]
is there any chance that you give me some proxies? i would pay for it if needed

**Comment by HerrEurobeat** [comment]
I'm using all of the proxies I'm currently paying for, sorry

**Comment by Yarumina** [comment]
okay no problem. i can use the same proxy for many accs right? and the bot is random picking proxies

**Comment by HerrEurobeat** [comment]
Yes, you can use one proxy for multiple accounts, the less accounts per proxy the better though.  
The bot won't pick random, it will spread all accounts over all available IPs evenly.

**Comment by Yarumina** [comment]
Okay i understand. Do i need the username and password or can i use only the ip and port

**Comment by HerrEurobeat** [comment]
Most proxies require authentication - so in order to use them yes. If your proxies don't, then no.

**Comment by Yarumina** [comment]
Do i only need to add the ip and port or http:// too?

**Comment by HerrEurobeat** [comment]
Not sure from the top of my head but including it won't hurt

**Comment by HerrEurobeat** [comment]
Since you didn't reply I am going to close this issue and hope that I answered your question.  
If not, feel free to reopen this issue or please create another issue if there is something else on your mind!
